crawler/src/test/java/com/brandwatch/minibcr/crawler/service/reddit/RedditCrawlerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.brandwatch.minibcr.crawler.service.reddit;
+
+import com.brandwatch.minibcr.crawler.model.reddit.SubReddit;
+import com.brandwatch.minibcr.crawler.service.reddit.auth.RedditAuthenticator;
+import com.brandwatch.minibcr.crawler.service.reddit.auth.RedditClient;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Collections;
+
+import static org.mockito.Mockito.times;
+
+@SpringBootTest
+@RunWith(MockitoJUnitRunner.class)
+public class RedditCrawlerTest {
+
+
+    @Mock
+    private RedditAuthenticator redditAuthenticator;
+
+    @Mock
+    private RestTemplate restTemplate;
+
+    @Mock
+    private RedditClient redditClient;
+
+    @Mock
+    private Producer producer;
+
+
+
+    @Test
+    public final void crawl() {
+        RedditCrawler redditCrawler = new RedditCrawler(redditAuthenticator,redditClient,restTemplate,producer);
```

Comment:
```suggestion
        RedditCrawler redditCrawler = new RedditCrawler(redditAuthenticator, redditClient, restTemplate, producer);
```
NABD, but linters in other repositories will shout for this sort of thing